src/rules/orderedImportsRule.ts

```diff
@@ -104,27 +116,39 @@ const TRANSFORMS = new Map<string, Transform>([
     ["case-insensitive", (x) => x.toLowerCase()],
     ["lowercase-first", flipCase],
     ["lowercase-last", (x) => x],
+    ["full-path", (x) => x],
+    ["module-name", (x) => {
+        const splitIndex = x.lastIndexOf("/");
```

Comment:
Should probably handle imports from node_modules special. Otherwise sorting would be odd when using submodule imports or scoped modules:
```ts
import foo from "a/foo"
import bar from "b/bar"
```

Reply:
@ajafff is there an easy way to tell when importing from node_modules vs. relative?

Reply:
Take a look at `noSubmoduleImportsRule.ts` it contains such a condition:
https://github.com/palantir/tslint/blob/7131f014e79a137d6b7a5bd0a7cad5126efe0c86/src/rules/noSubmoduleImportsRule.ts#L85

Reply:
Done :)